validphys2/src/validphys/process_options.py

```diff
@@ -229,8 +237,11 @@ def _hqp_mqq_xq2map(kin_info):
 
 def _inc_xq2map(kin_info):
     # Compute x, Q2
-    # k2 necessary to take the mass for DY inclusive cross sections still not migrated
-    mass2 = kin_info.get_one_of(_Vars.m_W2, _Vars.m_Z2, _Vars.m_t2, "k2")
+    try:
+        mass = kin_info.get_one_of(_Vars.m_W, _Vars.m_Z, _Vars.m_t, _Vars.m_ll)
```

Comment:
Why are the first three needed? AFAICT, there are no datasets (and there shouldn't) that are expressed in these (?).

Reply:
As I put in the OP, I think we should not use the squares (since they look much uglier in plots), since I was changing these files I thought I might as well prepare it.